rules/falco_rules.yaml

```diff
@@ -1153,6 +1153,9 @@
 - macro: etcd_manager_updating_dns
   condition: (container and proc.name=etcd-manager and fd.name=/etc/hosts)
 
+- macro: automount_using_mtab
+  condition: (proc.pname = automount and fd.name startswith /etc/mtab)
```

Comment:
```suggestion
  condition: (proc.pname=automount and fd.name=/etc/mtab)
```

WDYT?

Reply:
After having spent some hours with Nicolas on this we decided to merge PR as is.

He used `startswith` because he had `automount` daemon writing to `/etc/mtab.[something]` files (which to me seems something happening when the `/etc/mtab` file cannot be written because it's locked).

In the next days he's going to observe again the original event in order to understand whether it is better to use equal or startswith here.